featurebyte/schema/feature.py

```diff
@@ -34,6 +34,80 @@ class FeatureCreate(FeatureByteBaseModel):
     feature_namespace_id: Optional[PydanticObjectId] = Field(default_factory=ObjectId)
```

Comment:
`feature_namespace_id` should be removed from `FeatureCreate` payload (https://featurebyte.atlassian.net/browse/DEV-1758).